### PR TITLE
[Doc] Update Evaluation

### PR DESCRIPTION
**Pull request by tonysy**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -134,6 +134,32 @@ lmdeploy chat turbomind InternLM/internlm-chat-7b --model-name internlm-chat-7b
...
 **注意：**本项目中的全量训练功能已经迁移到了[InternEvo](https://github.com/InternLM/InternEvo)以便捷用户的使用。InternEvo 提供了高效的预训练和微调基建用于训练 InternLM 系列模型。
 
+
+## 评测
+
+我们使用 [OpenCompass](https://github.com/open-compass/opencompass) 进行模型评估。在 InternLM-2 中，我们主要标准客观评估、长文评估（大海捞针）、数据污染评估、智能体评估和主观评估。
+
+### 标准客观评测
+
+请按照 [OpenCompass 教程](https://github.com/open-compass/opencompass) 进行客观评测。我们通常在 **Base** 模型上使用 `ppl` 进行多项选择题，在 **Chat** 模型上使用 `gen` 进行所有问题。
```
Comment (BIGWangYuDong) [inline README_zh-CN.md:144]: 感觉描述有点奇怪

我们通常在 Base 模型上使用 ppl 进行多项选择题评测，在 Chat 模型上使用 gen 进行所有问题的答案生成和评测

File: README.md [hunk 2]
```diff
@@ -149,6 +149,32 @@ Please refer to [finetune docs](./finetune/) for fine-tuning with InternLM.
...
 **Note:** We have migrated the whole training functionality in this project to [InternEvo](https://github.com/InternLM/InternEvo) for easier user experience, which provides efficient pre-training and fine-tuning infra for training InternLM.
 
+
+## Evaluation
+
+We utilize [OpenCompass](https://github.com/open-compass/opencompass) for model evaluation. In InternLM-2, we primarily focus on standard objective evaluation, long-context evaluation (needle in a haystack), data contamination assessment, agent evaluation, and subjective evaluation.
+
+### Objective Evaluation
+
+To evaluate the InternLM model, please follow the guidelines in the [OpenCompass tutorial](https://github.com/open-compass/opencompass). Typically, we use `ppl` for multiple-choice questions on the **Base** model and `gen` for all questions on the **Chat** model.
```
Comment (BIGWangYuDong) [inline README.md:159]: URL may update to the tutorial link?

File: README.md [hunk 3]
```diff
@@ -77,6 +77,58 @@ The release of InternLM2 series contains two model sizes: 7B and 20B. 7B models
...
+| InternLM2 Chat 20B  | 21.75%   | 2373   |
+| Mixtral 8x7B v0.1  | 18.26%   | 1465   |
+| Claude 2            | 17.19%   | 1069   |
+| Gemini Pro         | 16.85%   | 1315   |
+| GPT-4 0613         | 15.76%   | 1140   |
+| Claude 2.1         | 15.73%   | 1096   |
+
+* According to the released performance of 2024-01-17.
+
+### Data Contamination
```
Comment (ZwwWayne) [inline README.md:122]: 感觉这个结果要不不放这儿了吧，以及是国内平均不是开源平均

**Review by ZwwWayne — COMMENTED** [review]
can be merged after fixing last comment

**Pull request merged**